pages/404.tsx

```diff
@@ -0,0 +1,20 @@
+import Link from 'next/link'
+import Image from 'next/image'
+
+export default function Custom404() {
+ return (
+     <>
+      <div className="w-9/12 m-auto py-8 min-h-screen flex items-center justify-center">
+	<Image src="/assets/png/404.jpg" alt="404 page image" width={1500} height={1000}

```

Comment:
check the image source once. Not loading in deployment preview. Probably /assets/... should be written as assets/...

Reply:
My mistake, I haven't added the image folder yet

Reply:
LGTM now.